docs/src/guides/hub-entities.md

```diff
@@ -0,0 +1,103 @@
+
+In ArcGIS Online, the item type is `Hub Site Application`
+
+In ArcGIS Enterprise, the item type is `Site Application`
+
+In both environments, we use Hub.js to work with sites via the `HubSiteManager`, which operates on `IHubSite` objects.
+
+#### API Links
+
+- [HubSiteManager (Not implemented yet)]()
+- [IHubSite](/hub.js/api/common/IHubSite)
+
+### Hub Pages
+
+A Hub Page is an item which represents a Page, which can be associated with one or more Hub Sites. It also has a [layout](/hub.js/api/common/IHubLayout/), but since it always renders in the context of a Hub Site, it does not have a theme, header or footer.
+
+In ArcGIS Online, the item type is `Hub Page`
+
+In ArcGIS Enterprise, the item type is `Site Page`
+
+In both environments, we use Hub.js to work with sites via the `HubPageManager`, which operates on `IHubPage` objects.
+
+#### API Links
+
+- [HubPageManager (Not implemented yet)]()
+- [IHubPage](/hub.js/api/common/IHubPage)
+
+### Hub Projects
+
+A Hub Project is a customizable representation of a real-world project. It can be associated with one or more Hub Sites or Hub Initiatives. It has a [banner image](), [timeline]() and can optinally have a [layout]().
+
+In ArcGIS Online, the item type is `Hub Project`
+
+Hub Projects are not available in ArcGIS Enterprise.
+
+We use Hub.js to work with Projects via the `HubProjectManager`, which operates on `IHubProject` objects.
+
+#### API Links
+
+- [HubProjectManager](/hub.js/api/common/HubProjectManager)
+- [IHubProject](/hub.js/api/common/IHubProject)
+
+### Hub Initiatives
+
+A Hub Initiative is a broad umbrella of a real-world project. It can be associated with a Hub Sites and one or more Hub Projects. It has a [banner image](), and can optinally have a [layout]().
+
+In ArcGIS Online, the item type is `Hub Initiative`
+
+Hub Initiatives are not available in ArcGIS Enterprise.
+
+We use Hub.js to work with Initiatives via the `HubInitiativeManager`, which operates on `IHubInitiative` objects.
+
+#### API Links
+
+- [HubInitiativeManager (Not implemented yet)]()
+- [IHubInitiative](/hub.js/api/common/IHubInitiative)
+
+### Hub Content
+
+Any other items surfaced in ArcGIS Hub, are abstracted into "Hub Content". Depending on the specific item type, the properties available may vary a lot - i.e. a multi-layer feature service exposes a lot more properties than a PDF Document.
+
+Within Hub we do allow users to edit some of the basic Item information - title, description, summary, thumbnail etc, and that is done via the `HubContentManager`, which operates on `IHubContent` objects.
+
+#### API Links
+
+- [HubContentManager (Not implemented yet)]()
+- [IHubContent](/hub.js/api/common/IHubContent)
+
+### Hub Events
+
+Hub Events are not currently backed by items, only available in ArcGIS Online (Hub Premium) and do not currently have a Manager class wrapper. The are represented by `IHubEvent`
```

Comment:
Can we still provide a manager interface? It shouldn't matter they're not items.